newt/newt.go

```diff
@@ -140,7 +140,9 @@ func newtCmd() *cobra.Command {
 		Run: func(cmd *cobra.Command, args []string) {
 			fmt.Printf("Apache Newt\n")
 			fmt.Printf("   Version: %s\n", newtutil.NewtVersionStr)
-			fmt.Printf("  Git Hash: %s\n", newtutil.NewtGitHash)
+			if newtutil.NewtGitHash != "unknown" {
+				fmt.Printf("  Git Hash: %s\n", newtutil.NewtGitHash)
```

Comment:
I don't have a strong feeling about this, but I think it is better to be explicit here.  I would just print "unknown" if we don't know the git hash.

Reply:
done!